src/cljs/main/broadfcui/common/table/column_editor.cljs

```diff
@@ -0,0 +1,148 @@
+(ns broadfcui.common.table.column-editor
+  (:require
+    [dmohs.react :as react]
+    [broadfcui.common :as common]
+    [broadfcui.common.components :as comps]
+    [broadfcui.common.icons :as icons]
+    [broadfcui.common.overlay :as overlay]
+    [broadfcui.common.table.utils :as table-utils]
+    [broadfcui.common.style :as style]
+    [broadfcui.utils :as utils]
+    ))
+
+
+(defn- set-all [{:keys [column-display update-column-display fixed-column-count]} visible?]
+  (->> (drop fixed-column-count column-display)
```

Comment:
Mohs and I decided that it only makes sense for unmovable columns to be the leftmost, so there's a `:fixed-column-count` property on the table now.  This is dropping the fixed columns from the logic that sets All/None visibility.